### PR TITLE
Cleaning up the README a bit

### PR DESCRIPTION
**Pull request by GreenGremlin**
Pulling over the README updates from https://github.com/apollographql/graphql-tag/pull/244/files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Helpful utilities for parsing GraphQL queries. Includes:
 
 ### gql
 
-This is a template literal tag you can use to concisely write a GraphQL query that is parsed into the standard GraphQL AST:
+The `gql` template literal tag can be used to concisely write a GraphQL query that is parsed into the standard GraphQL AST. It is the recommended method for passing queries to [Apollo Client](https://github.com/apollographql/apollo-client). While it is primaryily build for the Apollo client, it generates generic graphQL AST which could, potentially be used by any graphQL client.
```
Comment (dobesv) [inline README.md]: "primarily built for"

File: README.md [hunk 2]
```diff
@@ -64,78 +98,98 @@ That's where this package comes in - it lets you write your queries with [ES2015
...
-```js
-import query from './query.graphql';
+### Preprocessing queries and fragments
 
-console.log(query);
-// {
-//   "kind": "Document",
-// ...
-```
+Preprocessing graphQL queries and fragments to ASTs at built time can greatly increase load times.
```
Comment (dobesv) [inline README.md]: "at build time can greatly improve load times"

(I think load times will be less due to preprocessing, right?)

Comment (GreenGremlin) [reply README.md]: Yep! Fixed

**Comment by GreenGremlin** [comment]
Is there anything I can do to help get this merged?

**Comment by dobesv** [comment]
I don't have the power to merge unfortunately.  I'm not sure if the project maintainers have time to look at this or not.

**Pull request merged**